_sass/filter.scss

```
@@ -0,0 +1,291 @@
+        padding: 0 0.25rem;
+        & button {
+            padding: 0.5rem 0.1rem;
+            width: 10rem;
+            border-radius: 5px;
+            border: none;
+            cursor: pointer;
+        }
+    }
+  }
+  #x-icon{
+    display: inline-block;
```

Stylelint (17.14.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)